### ISSUE TITLE
Update accepted images

### ISSUE THREAD

**Issue by colliell** [Aerolyzer/Aerolyzer#79]
Accepted images in the config/image_restriction_conf.yaml should match what's on our FAQ page, need to change either FAQ page list of accepted images or accepted images in config

**Comment by kwhitehall** [comment]
Note that changing the yaml file requires a new release to be to date for the release. Nonetheless, I suggest updating the yaml file. When the new release is made, this will be reflected.

**Comment by colliell** [comment]
@kwhitehall makes sense, we will be fine for expo purposes but just wanted to make an issue for future reference (y)

**Comment by kwhitehall** [comment]
+1
